src/mon/AuthMonitor.cc

```diff
@@ -1276,7 +1276,14 @@ bool AuthMonitor::valid_caps(
     if (!moncap.parse(caps, out)) {
       return false;
     }
-  } else if (type == "mgr") {
+    return true;
+  }
+
+  if (!g_conf()->mon_auth_validate_all_caps) {
```

Comment:
```suggestion
  if (!g_conf().get_val<bool>("mon_auth_validate_all_caps")) {
```

Reply:
using this legacy form is a shortcut to avoid the config lock, you're right it's not needed here though